app/assets/javascripts/map.js

```diff
@@ -0,0 +1,44 @@
+$(function() {
+  var courtsRequest;
+  var map;
+
+  var createMap = function(latitude, longitude){
+    L.mapbox.accessToken = "pk.eyJ1IjoiZWxpa2ltIiwiYSI6Ik1YSm9TY3cifQ.-aONewTGq_GmSuvGS7xQaQ";
+    map = L.mapbox.map('map', 'elikim.kkgb4e2p').
+      setView([latitude, longitude], 14);
+  };
+
+  var createMarker = function(court) {
+    var marker = L.marker([ court.latitude, court.longitude], {
+      icon: L.mapbox.marker.icon({
+        'marker-color': '#9c89cc'
+      })
+    }).addTo(map);
+  };
+
+  var plotCourts = function(courts){
+    $.each(courts, function(_, court) {
+      createMarker(court);
+    });
+  };
+
+  if($("body").hasClass("courts-index")) {
```

Comment:
How do you feel about extracting a function that reveals intent a little better? Something like:

```javascript
if(isOnCourtsIndex()) {

}
else if(isOnCourtsShow()) {

}
```